modules/indigo/module/inc/indigo/of_state_manager.h

```diff
@@ -281,6 +303,56 @@ typedef struct indigo_core_gentable_ops {
         void *table_priv, void *entry_priv, of_list_bsn_tlv_t *key,
         of_desc_str_t err_txt);
 
+    /**
+     * @brief Add an entry
+     * @param op_ctx operation context 
+     * @param table_priv Table private data
```

Comment:
Please expand this explaining that table_priv should not be passed when returning PENDING, Also you can add that check in the code itself to return an error